app/views/prototypes/edit.html.haml

```diff
@@ -14,20 +14,20 @@
     .row
       .col-md-12
         = f.fields_for :prototype_images, (@prototype.prototype_images.main.first) do |main_image|
-          .cover-image-upload.main_image_preview
+          .cover-image-upload.main_image_preview{style: "background-image: url(#{@prototype.prototype_images.main.first.image});" "width: 100%"}
             = main_image.file_field :image
             = main_image.hidden_field :status, value: PrototypeImage.statuses[:main]
       .col-md-12
         %ul.proto-sub-list.list-group
           = f.fields_for :prototype_images, @prototype do |sub_image|
             %li.list-group-item.col-md-4
-              .image-upload.sub_image_preview1
+              .image-upload.sub_image_preview1{style: "background-image: url(#{@prototype.prototype_images.sub.first});"}
                 = sub_image.file_field :image
                 = sub_image.hidden_field :status, value: PrototypeImage.statuses[:sub]
 
           = f.fields_for :prototype_images, @prototype do |sub_image|
             %li.list-group-item.col-md-4
-              .image-upload.sub_image_preview2
+              .image-upload.sub_image_preview2{style: "background-image: url(#{@prototype.prototype_images.sub.first});"}
```

Comment:
Line is too long. [121/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace